api/login.ts

```diff
@@ -21,10 +21,14 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
 
     try {
       const message = new SiweMessage(data);
+
+      // TODO: replace by configured domain
+      // TODO: Include this in .env?
+      const domain = 'localhost:3000';
```

Comment:
@EsteFilipe can you update this to fetch the domain from the configuration file?